snappy/dpu_snappy.h

```diff
@@ -3,26 +3,22 @@
 
 #include "common.h"
 
-#define GET_ELEMENT_TYPE(_tag) (_tag & BITMASK(2))
+#define GET_ELEMENT_TYPE(_tag)  (_tag & BITMASK(2))
 #define GET_LENGTH_1_BYTE(_tag) ((_tag >> 2) & BITMASK(3))
 #define GET_OFFSET_1_BYTE(_tag) ((_tag >> 5) & BITMASK(3))
 #define GET_LENGTH_2_BYTE(_tag) ((_tag >> 2) & BITMASK(6))
 
 // Max length of the input and output files
-#define MAX_FILE_LENGTH 0x1000000
+#define MAX_FILE_LENGTH 0x2000000
```

Comment:
at some point, this should be redefined using the new MEGABYTE() macro from PIM-common (but not critical)

Reply:
Changed!